lib/stat_tracker.rb

```diff
@@ -131,6 +131,28 @@ def total_games_per_team_for(season_id)
     end
   end
 
+  def games_won_per_team 
+    @game_teams.reduce(Hash.new(0)) do |result, game_team| 
+      if game_team.result == "WIN" 
+        result[game_team.team_id] += 1 
+      else 
+        result[game_team.team_id] += 0
```

Comment:
I had to add += 0 here because I still needed to create the team_id key if a team hasn't won any games. Let me know if you have other ideas for how to better accomplish this!

Reply:
It feels like there should be a shorthand way to do this but I wasn't able to find anything promising online!

Reply:
Right!? I'll keep thinking about this as I used it again in another PR.

Reply:
Hmmm! I'll look into this as well!